source/query-with-natural-language/prompt-natural-language-query.txt

```diff
@@ -0,0 +1,113 @@
+.. _prompt-natural-language-query:
+
+========================================
+Prompt your First Natural Language Query
+========================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+
+You can use MongoDB Compass to generate filter queries using natural 
+language. Compass uses AI to generate filter queries based on prompts 
+you provide. Querying with natural language can be a helpful starting 
+point and assist you in learning to write MongoDB queries.
+
+About this Task
+---------------
+
+.. note::
+
+   .. include:: /includes/fact-natural-language-query.rst
+
+- You can query with natural language to create both filter queries and 
```

Comment:
How do you feel about dropping the `filter` word?
```suggestion
You can use MongoDB Compass to generate queries using natural 
language. Compass uses AI to generate queries based on prompts 
you provide. Querying with natural language can be a helpful starting 
point and assist you in learning to write MongoDB queries.

About this Task
---------------

.. note::

   .. include:: /includes/fact-natural-language-query.rst

- You can query with natural language to create both queries and 
```

Reply:
Makes sense!